doc_source/falco-fargate-ptrace-security.md

```diff
@@ -0,0 +1,119 @@
+# Securing Fargate Tasks with Falco Runtime Security
+
+ECS Fargate `1.4.0` [announced support](https://aws.amazon.com/about-aws/whats-new/2020/04/aws-fargate-launches-platform-version-14/) for the `CAP_SYS_PTRACE` linux capability.
+
+This new feature can be implemented with [Falco](falco.org), an open source runtime security project originally built by Sysdig, Inc and later donated to the Cloud Native Computing Foundation.
```

Comment:
```suggestion
This new feature can be implemented with [Falco](falco.org), an [open source runtime security project](https://github.com/falcosecurity/falco) originally built by Sysdig, Inc and later donated to the Cloud Native Computing Foundation.
```

I would personally link also the repository on GitHub for people willing to deep-dive into the code.